scripts/state/migrate-from-0.3.py

```diff
@@ -77,8 +77,10 @@
 q['records'] = new_records
 q['protocol_version'] = 4
 q['dynamic_resharding'] = False
-q['block_producers_per_shard'] = [50] * 8
-q['avg_fisherman_per_shard'] = [0] * 8
+# TODO please check this variable
```

Comment:
what do you mean here?

Reply:
I don't actually know for what this script is. It called `migrate` which may mean that it needs some previous naming. I'm not sure, that's why I've added TODO to check with reviewers.